ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -296,8 +296,8 @@ public void deleteRequestResponseTest() throws IOException {
     for (short version : versions) {
       short accountId = Utils.getRandomShort(random);
       short containerId = Utils.getRandomShort(random);
-      int deletionTimeMs =
-          (int) (SystemTime.getInstance().milliseconds() + random.nextInt((int) TimeUnit.DAYS.toMillis(100)));
+      long deletionTimeMs =
+          (SystemTime.getInstance().milliseconds() + random.nextInt((int) TimeUnit.DAYS.toMillis(100)));
```

Comment:
please use `Utils.getRandomLong()` instead of casting.

Also parenthesis is unnecessary